### PR TITLE
Temporarily disable IndexFeatureFile and tests to avoid creation of bad indices.

### PR DESCRIPTION
**Pull request by cmnbroad**
Not sure if there is a better way to do this. UserException seems wrong since its displays USER Error.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java [hunk 1]
```diff
@@ -33,7 +35,7 @@
         oneLineSummary = "Creates indices for Feature-containing files (eg VCF and BED files)",
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java]: Add in the text `(Disabled) ` at the start of both the `summary` and `oneLineSummary` here.

File: src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java [hunk 2]
```diff
@@ -33,7 +35,7 @@
...
         programGroup = VariantProgramGroup.class
 )
-@DocumentedFeature
+@BetaFeature
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java]: Add comment to restore `@DocumentedFeature` when the tool is fixed.

File: src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java [hunk 3]
```diff
@@ -51,6 +53,9 @@
 
     @Override
     protected Object doWork() {
+        if (true) {
+            throw new GATKException("The IndexFeatureFile tool is temporarily disabled.");
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java]: `UserException` -- it's their fault for trying to run a disabled tool! :)

File: src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java (hunk 1 above)
Comment (cmnbroad) [inline src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java (hunk 2 above)
Comment (cmnbroad) [inline src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java]: Done.

File: src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java (hunk 3 above)
Comment (cmnbroad) [inline src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java]: Done.

**Comment by magicDGS** [comment]
Is there any time-plan to revert this tool?

**Comment by cmnbroad** [comment]
@magicDGS I expect we'll try to get this done as quickly as we can, though it will require a fix in htsjdk.

**Comment by magicDGS** [comment]
Thanks @cmnbroad!

**Pull request merged**